AgileBoard.Api/test/AgileBoard.Api.Tests.Unit/UserStoriesControllerTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿using AgileBoard.Api.Controllers.V1;
+using AgileBoard.Api.Domain;
+using AgileBoard.Api.Enums;
+using AgileBoard.Api.Services;
+using AgileBoard.Api.Tests.Unit.Helpers;
+using AutoMapper;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AgileBoard.Api.Tests.Unit
+{
+    internal class UserStoriesControllerTests
+    {
+        private readonly UserStoriesController _sut;
+        private readonly Mock<IUserStoryService> _userStoryServiceMock = new();
+        private readonly Mock<IMapper> _mapperMock = new();
```

Comment:
You have to register Mapping that you created otherwise mapper doesn't know anything about that. Plus Controller tests are more of integration tests.